examples/differential_correction_coupled.py

```diff
@@ -0,0 +1,237 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Thu Oct 25 17:19:30 2019
+
+@author: Wenyang Lyu and Shibabrat Naik
+
+Script to compute unstable periodic orbits at specified energies for the 
+coupled quartic Hamiltonian using differential correction
+"""
+
+import numpy as np
+from scipy.integrate import solve_ivp
+import time
+
+import matplotlib as mpl
+from matplotlib import cm
+from mpl_toolkits.mplot3d import Axes3D
+import matplotlib.pyplot as plt
+mpl.rcParams['mathtext.fontset'] = 'cm'
+mpl.rcParams['mathtext.rm'] = 'serif'
+
+# This needs testing for installation 
+import sys
+sys.path.insert(0, './src/')
```

Comment:
Not a big fan of manipulating `sys.path`. As a quick fix, I recommend adding some `__init__.py` files so that relative imports can be used. Next step would be to add a `setup.py` file so this becomes a full package.